namespacecost.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"github.com/gorilla/mux"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+)
+
+func (s *server) GetCost(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	namespace := vars["namespace"]
+
+	log.Printf("loading namespace costs for %q", namespace)
+
+	params := url.Values{}
+	params.Add("namespace", namespace)
+	params.Add("window", "1d")
+	params.Add("aggregation", "namespace")
+
+	resp, err := http.Get(kubecostURL + "/model/aggregatedCostModel?" + params.Encode())
```

Comment:
```suggestion
	u, err := url.Parse(kubecostURL)
	if err != nil {
		s.error(w, r, err)
		return
	}
	
	u, err = u.Parse("model/aggregatedCostModel")
	if err != nil {
		s.error(w, r, err)
		return
	}
	
	params := u.Query()
	params.Add("namespace", namespace)
	params.Add("window", "1d")
	params.Add("aggregation", "namespace")
	u.RawQuery = params.Encode()

	resp, err := http.Get(u.String())
```

I have a preference for using the built-in objects for URL parsing then just string-appending. @brendangadd thoughts?

Reply:
@zachomedia Sure thing. In that case, I recommend:
* `url.Parse` the Kubecost URL at startup so we can:
  - Fail fast with any resulting `err`
  - Pass the URL object to dependent components rather than a string
  - Avoid parsing on every request
* Suppress `err` from the subsequent `Parse` for path concatenation; I don't think it's possible for that constant path to generate an error.
* Maintain the `params := url.Values{}` approach as `u.Parse("some/path")` will clobber any existing query (i.e. `u.Query()` will always be an empty `url.Values`).

/cc @saffaalvi